mlrun/datastore/helpers.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+ONE_GB = 1024 * 1024 * 1024
+ONE_MB = 1024 * 1024
```

Comment:
I guess it's ok to move anyway, but seems like you didn't use it in this PR in the end.

Reply:
I think it is also better to move it.